### PR TITLE
Add `autocomplete` attrs to input fields

### LINKED ISSUE #36945: [BITV] 9.1.3.5/6.1 - Input fields that clearly refer to the user, such as "Last Name," "First Name," etc., do not convey the purpose of each field through a language-independent attribute (currently, only the "autocomplete" attribute is adequately supported). (1)
See also https://html.spec.whatwg.org/multipage/forms.html "Enabling client-side automatic filling of form controls" and Section 7 under https://www.w3.org/TR/WCAG21/ "Input Purposes for User Interface Components" for more information. 

- [ ] - [ ] Fix
- [ ] Backport

ea791921f8d91989b62585d40226bf0f

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-2841f578da-v6-n1

</details>

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
* Resolves: https://github.com/nextcloud/server/issues/36945

## Summary

No visual changes.
Add `autocomplete` attrs to input fields. Additional info: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete and https://www.w3.org/TR/WCAG21/#input-purposes section "7. Input Purposes for User Interface Components"

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by artonge — APPROVED** [review]
:ok_hand:

File: apps/settings/src/components/PersonalInfo/EmailSection/Email.vue [hunk 1]
```diff
@@ -105,11 +105,13 @@ import {
 	updateAdditionalEmail,
 } from '../../../service/PersonalInfo/EmailService.js'
 import { validateEmail } from '../../../utils/validate.js'
+import AccountPropertySection from '../shared/AccountPropertySection.vue'
```
Comment (Pytal) [inline apps/settings/src/components/PersonalInfo/EmailSection/Email.vue:108]:
```suggestion
```
Unused

File: apps/settings/src/components/PersonalInfo/EmailSection/Email.vue [hunk 2]
```diff
@@ -105,11 +105,13 @@ import {
...
 
 export default {
 	name: 'Email',
 
 	components: {
+		AccountPropertySection,
```
Comment (Pytal) [inline apps/settings/src/components/PersonalInfo/EmailSection/Email.vue:114]:
```suggestion
```
Also unused

**Review by Pytal — APPROVED** [review]
Apart from removals requested below

File: apps/settings/src/components/PersonalInfo/EmailSection/Email.vue (hunk 1 above)
Comment (JuliaKirschenheuter) [reply apps/settings/src/components/PersonalInfo/EmailSection/Email.vue:108]: thank you!

**Pull request merged**